### PR TITLE
server: ensure /var/lib/ocid/images exists

### PR DESCRIPTION
**Pull request by runcom**
Fixes https://github.com/mrunalp/ocid/issues/28

@mrunalp @hmeng-19 PTAL

Signed-off-by: Antonio Murdaca runcom@redhat.com

### CODE REVIEW

File: server/server.go [hunk 1]
```diff
@@ -2,13 +2,15 @@ package server
...
 	"fmt"
+	"os"
 
 	"github.com/mrunalp/ocid/oci"
 	"github.com/mrunalp/ocid/utils"
 )
 
 const (
 	runtimeAPIVersion = "v1alpha1"
+	imageStore        = "/var/lib/ocid/images"
```
Comment (haiyanmeng) [inline server/server.go:13]: @runcom , @mrunalp , should we provide an option of `ocid` for setting this path, like `--sandboxDir` and `--containerDir`?

File: server/image.go [hunk 2]
```diff
@@ -52,11 +53,12 @@ func (s *Server) PullImage(ctx context.Context, req *pb.PullImageRequest) (*pb.P
 	if err != nil {
 		return nil, err
 	}
+
 	// TODO: make sure this dir exists?
```
Comment (haiyanmeng) [inline server/image.go]: @runcom , this `TODO` can be removed since you are fixing it with this PR.

File: server/server.go (hunk 1 above)
Comment (runcom) [inline server/server.go:13]: probably a follow up when integrating with containers/storage

File: server/image.go (hunk 2 above)
Comment (runcom) [inline server/image.go]: fixed

File: server/image.go [hunk 3]
```diff
@@ -52,11 +53,11 @@ func (s *Server) PullImage(ctx context.Context, req *pb.PullImageRequest) (*pb.P
 	if err != nil {
 		return nil, err
 	}
-	// TODO: make sure this dir exists?
-	if err := os.Mkdir("/var/lib/ocid/images/"+tr.StringWithinTransport(), 0755); err != nil {
+
+	if err := os.Mkdir(filepath.Join(imageStore, tr.StringWithinTransport()), 0755); err != nil {
```
Comment (haiyanmeng) [inline server/image.go:57]: @runcom , should we use `MkdirAll` here? If `/var/lib/ocid` does not exist, this will fail currently.

Comment (runcom) [inline server/image.go:57]: Top dir should be created on server initialization not here, I'll defer this to another PR

Comment (haiyanmeng) [inline server/image.go:57]: @runcom , sounds good.

**Comment by haiyanmeng** [comment]
@mrunalp , @runcom , I tested this PR, and it works well.

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**